app/src/main/java/com/arygm/quickfix/ui/authentication/LogInScreen.kt

```diff
@@ -0,0 +1,263 @@
+                          contentDescription = null,
+                          contentScale = ContentScale.Crop,
+                          alignment = Alignment.TopStart,
+                          modifier = Modifier.fillMaxWidth().size(180.dp))
+
+                      QuickFixBackButtonTopBarLogin(
+                          onBackClick = {
+                            shrinkBox = false
+                            navigationActions.goBack()
+                          },
+                          color = Color.Transparent,
+                      )
+                    }
+
+                // Foreground content (on top of the TopAppBar)
+                Box(
+                    modifier =
+                        Modifier.fillMaxWidth()
+                            .align(Alignment.BottomStart)
+                            .zIndex(2f)
+                            .background(
+                                colorScheme.background,
+                                shape =
+                                    RoundedCornerShape(12.dp)) // Ensure content is above TopAppBar
+                    ) {
+                      Box(
+                          modifier =
+                              Modifier.align(Alignment.BottomStart)
+                                  .size(180.dp, 180.dp)
+                                  .offset(x = (-150).dp, y = 64.dp)
+                                  .graphicsLayer(rotationZ = 57f)
+                                  .background(colorScheme.primary)
+                                  .testTag("BoxDecoration"))
+
+                      Column(
+                          modifier =
+                              Modifier.align(Alignment.Center)
+                                  .padding(16.dp)
+                                  .zIndex(100f), // Ensure it's on top
+                          horizontalAlignment = Alignment.CenterHorizontally,
+                          verticalArrangement = Arrangement.Center) {
+                            Text(
+                                "Login",
+                                style = MaterialTheme.typography.headlineLarge,
+                                color = colorScheme.primary,
+                                modifier = Modifier.testTag("WelcomeText"))
+
+                            Spacer(modifier = Modifier.padding(3.dp))
+
+                            Text(
+                                "Your perfect fix is just a click away !",
+                                style = MaterialTheme.typography.labelSmall,
+                                color = colorScheme.onSecondaryContainer,
+                                modifier = Modifier.testTag("WelcomeTextBis"))
+
+                            Spacer(modifier = Modifier.padding(10.dp))
+
+                            QuickFixTextField(
+                                value = email,
+                                onValueChange = { email = it },
+                                label = "Username or Email",
```

Comment:
I don't think we will have support for a `Username` field. I am not sure how I feel about a painter with username "NoobSlayer1987" being commissioned to work on my walls 😅 
```suggestion
                                label = "Email",
```

Reply:
I believe we should give users/workers the option to log in with a username, which could be their display name if they log in via Google or simply a unique username. Having a username field in each profile would also make database management easier, as we wouldn’t rely only on email addresses. This could also allow for future features, like linking multiple emails (personal and work). So, enabling login with a username is essential in my opinion.

We can talk about it later today at the meeting 😄

Reply:
This is a fair point. Resolving this conversation :)